Resources/Machine Learning/qml/mlClassificationBoosting.qml

```diff
@@ -126,114 +127,8 @@ Form {
-                id: holdoutManual
-                name: "holdoutManual"
-                childrenOnSameRow: true
-                text: qsTr("Sample")
-
-                RowLayout {
-                
-                    PercentField {    
-                        name: "testDataManual"
-                        defaultValue: 20
-                        min: 5
-                        max: 95 
-                    }
-
-                    Text {
-                        text: qsTr("of all data")
-                        enabled: true
-                    }
-                }
-            }
-
-            CheckBox { 
-                id: addIndicator  
-                name: "addIndicator"
-                text: qsTr("Add generated indicator to data")
-                Layout.leftMargin: 20
-                enabled: holdoutManual.checked
-
-                ComputedColumnField { 
-                    name: 		"testIndicatorColumn"
-                    text: 		"Name: "
-                    fieldWidth: 120
-                    visible:    addIndicator.checked
-                }
-            }
-
-            RadioButton {
-                id: testSetIndicator
-                name: "testSetIndicator"
-                label: qsTr("Test set indicator:")
-                childrenOnSameRow: true
-
-                DropDown {
-                    name: "testSetIndicatorVariable"
-                    showVariableTypeIcon: true
-                    addEmptyValue: true
-                    placeholderText: qsTr("None")
-                }
-            }
-        }
-
-        RadioButtonGroup {
-            title: qsTr("Training and Validation Data")
-            name: "modelValid"
-
-            RadioButton {
-                name: "validationManual"
-                childrenOnSameRow: true
-                checked: true
-                text: qsTr("Sample")
-
-                RowLayout {
-
-                    PercentField {     
-                        name: "validationDataManual"
-                        defaultValue: 20
-                        min: 5
-                        max: 95
-                    }
-
-                    Text {
-                        text: qsTr("for validation data")
-                        enabled: true
-                    }
-                }
-            }
     
-            RadioButton { 
-                name: "validationKFold"
-                childrenOnSameRow: true
-                text: qsTr("K-fold with")
-
-                RowLayout {
-
-                    IntegerField {
-                        name: "noOfFolds"
-                        defaultValue: 5
-                        min: 2
-                        max: 999
-                        fieldWidth: 30
-                    } 
-
-                    Text {
-                        text: qsTr("folds")
-                        enabled: true
-                    }
-                }
-            }
-        }
-    }
+    ML.DataSplit {leaveOneOutVisible: false}
```

Comment:
Maybe add some spacing to make it more readable and consistent with all the other qml files and code